crates/primitives/src/types.rs

```diff
@@ -323,6 +323,7 @@ pub trait Environment: Clone {
     ///
     /// The value must match the maximum number of supported event topics of the used
     /// runtime.
+    // TODO: (@davidsemakula) Remove, no longer configurable in `pallet-revive`.
```

Comment:
There's also a bunch of places in `cargo-contract` where we need to remove it.

Reply:
Gotcha, will add it to the list for the follow up issue